### PR TITLE
fix Navigation button without HTML <a> tag

### PR DESCRIPTION
**Pull request by Sokan93**

### CODE REVIEW

**Comment by Sokan93** [comment]
> Seems to work.
> 
> However some Cypress tests needs to be updated. See breaking E2E tests.

Please how do I go about resolving the breaking E2E tests?

**Comment by znarf** [comment]
Look at the results:

- https://circleci.com/gh/opencollective/opencollective-frontend/31259
- https://circleci.com/gh/opencollective/opencollective-frontend/31258

You need to edit the tests to match the updated HTML markup.

- https://github.com/opencollective/opencollective-frontend/blob/master/cypress/integration/01-collective.test.js#L31
- https://github.com/opencollective/opencollective-frontend/blob/master/cypress/integration/27-expenses.create.test.js#L24

**Comment by Sokan93** [comment]
> Look at the results:
> 
> * https://circleci.com/gh/opencollective/opencollective-frontend/31259
> * https://circleci.com/gh/opencollective/opencollective-frontend/31258
> 
> You need to edit the tests to match the updated HTML markup.
> 
> * https://github.com/opencollective/opencollective-frontend/blob/master/cypress/integration/01-collective.test.js#L31
> * https://github.com/opencollective/opencollective-frontend/blob/master/cypress/integration/27-expenses.create.test.js#L24

I updated the tests files but it still fails during the testing.

**Comment by znarf** [comment]
Maybe this is not the right fix? Inspect the HTML markup and figure the right selector.

Look at the deployment preview: https://opencollective-frontend-rewnrbdltf.now.sh/babel

File: src/components/MenuBar.js [hunk 1]
```diff
@@ -326,9 +326,9 @@ class MenuBar extends React.Component {
         </style>
         {this.state.sticky && stickyCTA && cta}
         {!collective.isArchived && ['COLLECTIVE', 'EVENT'].indexOf(collective.type) !== -1 && (
-          <Link className="submitExpense darkBackground" href={`${collective.path}/expenses/new`}>
```
Comment (znarf) [inline src/components/MenuBar.js]: No, this was working. Why change?

Comment (Sokan93) [reply src/components/MenuBar.js]: The selector I got when I inspected was different, or should I still leave it using <Link>?

Comment (znarf) [reply src/components/MenuBar.js]: Yes, it was fine with `<Link>`.

You should not confuse the React source code and the HTML generated code.

In this case, the React `<Link>` creates an HTML `<a>`.

**Comment by znarf** [comment]
Your last commit seems wrong, you're confusing React and  HTML.

The test are using HTML/CSS selectors, not React, so it's `a` that you need to use here.

File: cypress/integration/27-expenses.create.test.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const uploadReceipt = (dropzoneElement = '.InputTypeDropzone') => {
 describe('new expense when logged out', () => {
   it('requires to login to submit an expense', () => {
     cy.visit('/testcollective/expenses');
-    cy.get('.Button.submitExpense.darkBackground')
+    cy.get('.a.submitExpense.darkBackground')
```
Comment (znarf) [inline cypress/integration/27-expenses.create.test.js]: Here, you don't want a "class selector" `.a` but an "element selector" `a`.

Comment (Sokan93) [reply cypress/integration/27-expenses.create.test.js]: Thanks a lot, @znarf. Updating corrections

Comment (Sokan93) [reply cypress/integration/27-expenses.create.test.js]: Please, I'm getting a new error.

Comment (znarf) [reply cypress/integration/27-expenses.create.test.js]: I'm restarting the job, maybe a random error, doesn't seems related with the changes.

**Comment by znarf** [comment]
I rebased the branch and also merged the commits together. https://github.com/opencollective/opencollective-frontend/pull/2136

I don't know what was breaking the tests, but they're now passing. We will merge that tomorrow.

**Comment by Sokan93** [comment]
> I rebased the branch and also merged the commits together. #2136
> 
> I don't know what was breaking the tests, but they're now passing. We will merge that tomorrow.

Alright. Great!
Thank you

**Pull request closed without merging**